app/experimenter/experiments/forms.py

```diff
@@ -297,6 +299,35 @@ def clean(self):
                     )
 
 
+class CustomModelChoiceIterator(ModelChoiceIterator):
+
+    def __iter__(self):
+        yield (
+            CustomModelMultipleChoiceField.ALL_KEY,
+            CustomModelMultipleChoiceField.ALL_LABEL,
+        )
+        for choice in super().__iter__():
+            yield choice
+
+
+class CustomModelMultipleChoiceField(forms.ModelMultipleChoiceField):
+    """Return a ModelMultipleChoiceField but with the exception that
+    there's one extra "All" choice inserted as the first choice.
+    And when submitted, if "All" was one of the choices, reset
+    it to chose nothing."""
+
+    ALL_KEY = "__all__"
+    ALL_LABEL = "ALL"
```

Comment:
Instead of `ALL` change this to `All Locales` and `All Countries` and pass it in as an init param for the field?

Reply:
Done. 
<img width="847" alt="Screen Shot 2019-03-15 at 8 14 40 AM" src="https://user-images.githubusercontent.com/26739/54430620-8ba31580-46fa-11e9-8dea-b8a0573f5f6f.png">